portfolio/src/main/java/com/google/sps/servlets/SentimentAnalysis.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet which processes request to analyze message. */
+@WebServlet("/sentiment")
+public class SentimentAnalysis extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String message;
+    
+    // Check for the message parameter in request.
+    try {
+      message = request.getParameter("message");
+    } catch(Exception e) {
+      message = "";
```

Comment:
I think you can create a variable like this `String message = new String()` and it will be already initialized as an empty string:
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#String%28%29

Reply:
Isn't line 
String message;
interpreted as 
String message = new String();? 

I thought that it is the same as in C++(